label_studio/ml_models/models.py

```diff
@@ -75,3 +76,52 @@ class ThirdPartyModelVersion(ModelVersion):
 
     def has_permission(self, user):
         return user.active_organization == self.organization
+
+
+class ModelRun(models.Model):
+    class ProjectSubset(models.TextChoices):
+        ALL = 'All', _('All')
+        HASGT = 'HasGT', _('HasGT')
+
+    class FileType(models.TextChoices):
+        INPUT = 'Input', _('Input')
+        OUTPUT = 'Output', _('Output')
+
+    class ProjectStatus(models.TextChoices):
+        PENDING = 'Pending', _('Pending')
+        COMPLETED = 'Completed', ('Completed')
+        FAILED = 'Failed', ('Failed')
+        CANCELED = 'CANCELED', ('CANCELED')
+
+    organization = models.ForeignKey(
+        'organizations.Organization', on_delete=models.CASCADE, related_name='model_runs', null=True
+    )
+
+    project = models.ForeignKey(Project, on_delete=models.CASCADE, related_name='model_runs')
+
+    model_version = models.ForeignKey(ThirdPartyModelVersion, on_delete=models.CASCADE, related_name='model_runs')
+
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='model_runs',
+        on_delete=models.SET_NULL,
+        null=True,
+    )
+
+    project_subset = models.CharField(max_length=255, choices=ProjectSubset.choices, default=ProjectSubset.HASGT)
+    status = models.CharField(max_length=255, choices=ProjectStatus.choices, default=ProjectStatus.PENDING)
+
+    created_at = models.DateTimeField(_('created at'), auto_now_add=True)
+
+    triggered_at = models.DateTimeField(_('triggered at'))
+
+    completed_at = models.DateTimeField(_('completed at'), null=True, default=None)
+
+    # todo may need to clean up in future
+    @property
+    def get_input_file_name(self):
```

Comment:
These should either not be `@property` and be called `get_`* , or have the decorator and be called `input_file_name` for example

Reply:
actually I meant it to be input_file_name will update it

Reply:
done